test/CMakeLists.txt

```diff
@@ -32,6 +32,22 @@ else()
     message(STATUS "oneDPL: -fno-fast-math is not supported by current compiler")
 endif()
 
+# usage of Kernel names in tests
+set(TestExplicitKernelNameVariants AUTO ALWAYS)
```

Comment:
What do you think about having "ON" and "OFF" options as some other existing configuration variables have? You can try https://cmake.org/cmake/help/v3.4/module/CMakeDependentOption.html#module:CMakeDependentOption.

Reply:
You can try:
```
include(CMakeDependentOption)
cmake_dependent_option(TEST_EXPLICIT_KERNEL_NAMES "Explicitly providing kernel names in the tests" ON "ONEDPL_USE_UNNAMED_LAMBDA" OFF)
```

Reply:
Are we able in this case setup this property value in CMake params too ?

Reply:
I tested this command and it looks it does not support overriding. So a user should solely rely on the dependecy('ONEDPL_USE_UNNAMED_LAMBDA), which does not suit us.